tests/integration/test_alert_rules.py

```diff
@@ -1,6 +1,8 @@
 import pytest
+from tests.integration.conftest import append_profile
```

Comment:
Since this gets imported everywhere, could include it on the command_runner itself, or subclass the runner to be `command_runner_with_profile`